features/adres.feature

```diff
@@ -95,17 +100,16 @@ Functionaliteit: Adresvelden vullen
     - Wanneer de woonplaats geen waarde heeft, wordt de omschrijving van de gemeente van inschrijving als woonplaats gebruikt
 
     Abstract Scenario: adresregel2 voor een binnenlandse verblijfplaats met <omschrijving>
-      Gegeven het systeem heeft een persoon met de volgende gegevens
+      Gegeven de persoon met burgerservicenummer '<burgerservicenummer>' heeft de volgende 'verblijfplaats' gegevens
       | naam                                 | waarde                |
-      | burgerservicenummer                  | <burgerservicenummer> |
       | gemeente van inschrijving (09.10)    | 0518                  |
-      | gemeenteVanInschrijving.omschrijving | 's-Gravenhage         |
-      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      En de 'verblijfplaats' heeft de volgende 'adres' gegevens
       | naam                   | waarde           |
       | straatnaam (11.10)     | een straat       |
       | huisnummer (11.20)     | 1                |
       | postcode (11.60)       | <postcode>       |
       | woonplaatsnaam (11.70) | <woonplaatsnaam> |
```

Comment:
woonplaatsnaam (11.70) komt niet voor in de columnNameMap. Toevoegen of woonplaats (11.70) gebruiken

Reply:
In de propertyNameMap kwam hij wel voor. Ik had hem lokaal toegevoegd, maar die step_defs.js niet meegenomen met de PR. Is het geen probleem als hetzelfde veld in propertyNameMap woonplaatsnaam (11.70) heet terwijl het in columnNameMap woonplaats (11.70 heet ? 

Voor deze feature pas ik hem aan naar woonplaats (11.70) --> insert werkt. Scenario faalt nog wel.